addons/common/XEH_preInit.sqf

```diff
@@ -37,4 +37,22 @@ GVAR(addons) = _addons;
 // BWC
 #include "backwards_comp.sqf"
 
+// fix changing direction of remote units not working with zeus
+["ModuleCurator_F", "init", {
+    params ["_logic"];
+
+    _logic addEventHandler ["CuratorObjectEdited", {
+        params ["", "_unit"];
+
+        if (!local _unit) then {
+            private _dir = getDir _unit;
+            [_unit, _dir] remoteExec ["setDir", _unit];
```

Comment:
With doWatch you can also let tanks turn the gun. Would be nice to have ;D

Reply:
The gun turns when you rotate them anyway?

Reply:
The turret is rotating fine with this fix without having to do doWatch.